SimTracker/TrackerMaterialAnalysis/plugins/dd4hep/DD4hep_Listgroups.cc

```diff
@@ -257,7 +257,8 @@ void DD4hep_ListGroups::analyze(const edm::Event &evt, const edm::EventSetup &se
     bool firstChild = fv1.firstChild();
 
     for (const auto j : fv1.specpars()) {
-      for (const auto k : j->paths) {
+      for (const auto &k : j->paths) {
+        std::cout << k << std::endl;
```

Comment:
@smuzaffar are you adding this line on purpose?

Reply:
No it was a mistake to keep this line. I have fixed it. thanks for extra pairs of eyes.